### ISSUE TITLE
fixing warnings during TPTP conversion

### ISSUE THREAD

**Issue by arademaker** [ontologyportal/sumo#163]
I am trying to reproduce the banana slug reasoning reported in [1].

[1]	A. Pease, G. Sutcliffe, N. Siegel, and S. Trac, “Large Theory Reasoning with SUMO at CASC,” pp. 1–8, Jul. 2009.

Using the `SUMOKBtoTPTPKB` conversion. The conversion reveals the following errors:

```
$ java -Xmx7g -classpath $SIGMA_CP com.articulate.sigma.trans.SUMOKBtoTPTPKB
Info in KBmanager.initializeOnce()
Info in KBmanager.initializeOnce(): initializing with /Users/ar/workspace/KBs
KBmanager.readConfiguration()
KBmanager.serializedExists(): false
Info in KBmanager.initializeOnce(): reading from sources
Info in KBmanager.setConfiguration():
KBmanager.loadKB(): add constituent /Users/ar/workspace/sumo/Merge.kif to SUMO
INFO in KB.addConstituent(): /Users/ar/workspace/sumo/Merge.kif

INFO in KB.addConstituent(): added 6137 formulas and 1198 terms.
KBmanager.loadKB(): add constituent /Users/ar/workspace/sumo/Mid-level-ontology.kif to SUMO
INFO in KB.addConstituent(): /Users/ar/workspace/sumo/Mid-level-ontology.kif

INFO in KB.addConstituent(): added 7906 formulas and 2825 terms.
KBmanager.loadKB(): add constituent /Users/ar/workspace/sumo/banana.kif to SUMO
INFO in KB.addConstituent(): /Users/ar/workspace/sumo/banana.kif
..INFO in KB.addConstituent(): added 5 formulas and 13 terms.
INFO in KBcache.writeCacheFile(): SUMO_Cache.kif
INFO in KB.addConstituent(): /Users/ar/workspace/sumo/SUMO_Cache.kif

INFO in KB.addConstituent(): added 20399 formulas and 3214 terms.
INFO in KBcache.buildCaches(): size: 1021
INFO in KB.checkArity(): Performing Arity CheckINFO in PredVarInst.hasCorrectArityRecurse(): Predicate termporallyOverlaps does not have an arity defined in KB, can't get the arity number!
(termporallyOverlaps ?T
  (WhenFn ?P))
INFO in PredVarInst.hasCorrectArityRecurse(): Predicate approximateDiameter does not have an arity defined in KB, can't get the arity number!
(approximateDiameter ?Particle
  (MeasureFn ?Size Millimeter))
INFO in PredVarInst.hasCorrectArityRecurse(): Predicate shape does not have an arity defined in KB, can't get the arity number!
(shape ?OBJ Flat)
INFO in PredVarInst.hasCorrectArityRecurse(): Predicate routeInSystem does not have an arity defined in KB, can't get the arity number!
(routeInSystem ?P ?S)
INFO in PredVarInst.hasCorrectArityRecurse(): Predicate minValue does not have an arity defined in KB, can't get the arity number!
(minValue ?R ?ARG ?N)
INFO in PredVarInst.hasCorrectArityRecurse(): Predicate source does not have an arity defined in KB, can't get the arity number!
(source ?EMIT ?S)
INFO in PredVarInst.hasCorrectArityRecurse(): Predicate defaultMinValue does not have an arity defined in KB, can't get the arity number!
(defaultMinValue ?R ?ARG ?N)
INFO in PredVarInst.hasCorrectArityRecurse(): Predicate defaultMaxValue does not have an arity defined in KB, can't get the arity number!
(defaultMaxValue ?R ?ARG ?N)
INFO in PredVarInst.hasCorrectArityRecurse(): Predicate maxValue does not have an arity defined in KB, can't get the arity number!
(maxValue ?R ?ARG ?N)
INFO in PredVarInst.hasCorrectArityRecurse(): Predicate defaultValue does not have an arity defined in KB, can't get the arity number!
(defaultValue ?R ?ARG ?N)

INFO in KB.loadEProver(): Creating new process
INFO in KB.loadEProver(): generating TPTP file
.Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 1 of relation altitude in formula:
(altitude ?OBJ1 ?OBJ2 ?HEIGHT)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 2 of relation altitude in formula:
(altitude ?OBJ1 ?OBJ2 ?HEIGHT)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 3 of relation altitude in formula:
(altitude ?OBJ1 ?OBJ2 ?HEIGHT)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 1 of relation altitude in formula:
(altitude ?OBJ1 ?OBJ2 ?HEIGHT)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 2 of relation altitude in formula:
(altitude ?OBJ1 ?OBJ2 ?HEIGHT)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 3 of relation altitude in formula:
(altitude ?OBJ1 ?OBJ2 ?HEIGHT)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 1 of relation altitude in formula:
(altitude ?OBJ1 ?OBJ2 ?Q)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 2 of relation altitude in formula:
(altitude ?OBJ1 ?OBJ2 ?Q)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 3 of relation altitude in formula:
(altitude ?OBJ1 ?OBJ2 ?Q)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 1 of relation superficialPart in formula:
(superficialPart ?PART ?OBJ)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 2 of relation superficialPart in formula:
(superficialPart ?PART ?OBJ)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 1 of relation superficialPart in formula:
(superficialPart ?ELEMENT ?OBJ)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 2 of relation superficialPart in formula:
(superficialPart ?ELEMENT ?OBJ)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 1 of relation superficialPart in formula:
(superficialPart ?ELEMENT ?OBJ)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 2 of relation superficialPart in formula:
(superficialPart ?ELEMENT ?OBJ)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 1 of relation superficialPart in formula:
(superficialPart ?S ?B)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 2 of relation superficialPart in formula:
(superficialPart ?S ?B)
...Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 1 of relation superficialPart in formula:
(superficialPart ?PART ?OBJ)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 2 of relation superficialPart in formula:
(superficialPart ?PART ?OBJ)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 1 of relation superficialPart in formula:
(superficialPart ?PART ?OBJ)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 2 of relation superficialPart in formula:
(superficialPart ?PART ?OBJ)
.Error in FormulaPreprocessor.findType(): no type information for predicate routeInSystem
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 1 of relation routeInSystem in formula:
(routeInSystem ?P ?S)
Error in FormulaPreprocessor.findType(): no type information for predicate routeInSystem
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 2 of relation routeInSystem in formula:
(routeInSystem ?P ?S)
.Error in FormulaPreprocessor.findType(): no type information for predicate maxValue
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 1 of relation maxValue in formula:
(maxValue ?R ?ARG ?N)
Error in FormulaPreprocessor.findType(): no type information for predicate maxValue
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 2 of relation maxValue in formula:
(maxValue ?R ?ARG ?N)
Error in FormulaPreprocessor.findType(): no type information for predicate maxValue
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 3 of relation maxValue in formula:
(maxValue ?R ?ARG ?N)
Error in FormulaPreprocessor.findType(): no type information for predicate minValue
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 1 of relation minValue in formula:
(minValue ?R ?ARG ?N)
Error in FormulaPreprocessor.findType(): no type information for predicate minValue
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 2 of relation minValue in formula:
(minValue ?R ?ARG ?N)
Error in FormulaPreprocessor.findType(): no type information for predicate minValue
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 3 of relation minValue in formula:
(minValue ?R ?ARG ?N)
Error in FormulaPreprocessor.findType(): no type information for predicate approximateDiameter
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 1 of relation approximateDiameter in formula:
(approximateDiameter ?Particle
  (MeasureFn ?Size Millimeter))
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 1 of relation superficialPart in formula:
(superficialPart ?SP ?O)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 2 of relation superficialPart in formula:
(superficialPart ?SP ?O)
.Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 1 of relation superficialPart in formula:
(superficialPart ?PART1 ?OBJ)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 2 of relation superficialPart in formula:
(superficialPart ?PART1 ?OBJ)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 1 of relation superficialPart in formula:
(superficialPart ?PART2 ?OBJ)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 2 of relation superficialPart in formula:
(superficialPart ?PART2 ?OBJ)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 1 of relation superficialPart in formula:
(superficialPart ?PART1 ?OBJ)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 2 of relation superficialPart in formula:
(superficialPart ?PART1 ?OBJ)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 1 of relation superficialPart in formula:
(superficialPart ?PART2 ?OBJ)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 2 of relation superficialPart in formula:
(superficialPart ?PART2 ?OBJ)
..Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 1 of relation superficialPart in formula:
(superficialPart ?OBJ3
  (HoleHostFn ?HOLE))
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 1 of relation superficialPart in formula:
(superficialPart ?OBJ3
  (HoleHostFn ?HOLE))
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 1 of relation superficialPart in formula:
(superficialPart ?OBJ3
  (HoleHostFn ?HOLE))
..Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 1 of relation superficialPart in formula:
(superficialPart ?COVER ?BODY)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 2 of relation superficialPart in formula:
(superficialPart ?COVER ?BODY)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 1 of relation superficialPart in formula:
(superficialPart ?COVER ?BODY)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 2 of relation superficialPart in formula:
(superficialPart ?COVER ?BODY)
...Error in FormulaPreprocessor.findType(): no type information for predicate shape
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 1 of relation shape in formula:
(shape ?OBJ Flat)
....Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 1 of relation superficialPart in formula:
(superficialPart ?PART ?OBJ2)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 2 of relation superficialPart in formula:
(superficialPart ?PART ?OBJ2)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 1 of relation superficialPart in formula:
(superficialPart ?PART ?OBJ2)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 2 of relation superficialPart in formula:
(superficialPart ?PART ?OBJ2)
.Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 1 of relation superficialPart in formula:
(superficialPart ?OBJ1 ?OBJ2)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 2 of relation superficialPart in formula:
(superficialPart ?OBJ1 ?OBJ2)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 1 of relation superficialPart in formula:
(superficialPart ?PART ?WHOLE)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 2 of relation superficialPart in formula:
(superficialPart ?PART ?WHOLE)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 1 of relation superficialPart in formula:
(superficialPart ?OBJ3 ?OBJ2)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 2 of relation superficialPart in formula:
(superficialPart ?OBJ3 ?OBJ2)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 1 of relation superficialPart in formula:
(superficialPart ?OBJ3 ?OBJ2)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 2 of relation superficialPart in formula:
(superficialPart ?OBJ3 ?OBJ2)
..............................................................................................................................................................................................................................................................................................................................
INFO in EProver.writeBatchFile(): writing EBatchConfig.txt with KB file /Users/ar/workspace/sumo/SUMO.tptp
INFO in EProver(): executable: /Users/ar/workspace/E/PROVER/e_ltb_runner
INFO in EProver(): kbFile: /Users/ar/workspace/sumo/SUMO.tptp
EProver(): command: [/Users/ar/workspace/E/PROVER/e_ltb_runner, --interactive, /Users/ar/workspace/sumo/EBatchConfig.txt, /Users/ar/workspace/E/PROVER/eprover]
EProver(): new process: java.lang.UNIXProcess@685f4c2e
kbsFromXML(): Completed loading KBs
Info in KBmanager.setConfiguration(): Using kbDir: /Users/ar/workspace/sumo
NLGUtils.init(): initializing with /Users/ar/workspace/sumo
NLGUtils.readKeywordMap():
NLGUtils.serializedExists(): false
INFO in NLGUtils.readKeywordMap(/Users/ar/workspace/sumo/Translations/language.txt)
Filling keywordMap
EXIT NLGUtils.readKeywordMap(/Users/ar/workspace/sumo) with size 56
NLGUtils.serialize(): nlg size 56
NLGUtils.serialize(): NLGUtils has been serialized: com.articulate.sigma.nlg.NLGUtils@728938a9
WordNet.initOnce(): using baseDir = /Users/ar/workspace/KBs/WordNetMappings
WordNet.loadFresh():
INFO in WordNet.compileRegexPatterns(): compiling patterns
INFO in WordNet.readNouns(): Reading WordNet noun files
...............
Error in WordNet.readNouns(): No match in /Users/ar/workspace/sumo/WordNetMappings/WordNetMappings30-noun.txt for line ElectricalConductor+
...................................................................x
  2.396 seconds to process /Users/ar/workspace/sumo/WordNetMappings/WordNetMappings30-noun.txt with 82242 lines
  0.004 seconds to process /Users/ar/workspace/sumo/WordNetMappings/noun.exc
INFO in WordNet.readVerbs(): Reading WordNet verb files
.............x
  0.793 seconds to process /Users/ar/workspace/sumo/WordNetMappings/WordNetMappings30-verb.txt with 13837 lines
  0.004 seconds to process /Users/ar/workspace/sumo/WordNetMappings/verb.exc
INFO in WordNet.readAdjectives(): Reading WordNet adjective files
..................x
  0.398 seconds to process /Users/ar/workspace/sumo/WordNetMappings/WordNetMappings30-adj.txt with 18226 lines
INFO in WordNet.readAdverbs(): Reading WordNet adverb files
...x
  0.086 seconds to process /Users/ar/workspace/sumo/WordNetMappings/WordNetMappings30-adv.txt with 3691 lines
INFO in WordNet.readWordFrequencies(): Reading WordNet word frequencies
.................................x
  1.138 seconds to process /Users/ar/workspace/sumo/WordNetMappings/wordFrequencies.txt
INFO in WordNet.readStopWords(): Reading stop words
  0.001 seconds to process /Users/ar/workspace/sumo/WordNetMappings/stopwords.txt
INFO in WordNet.readSenseIndex(): Reading WordNet sense index
..............................................................................................................................................................................................................x
  2.068 seconds to process /Users/ar/workspace/sumo/WordNetMappings/index.sense with 206000 senses.
INFO in WordNet.readSenseCount(): Reading WordNet sense counts
.....Info in WordNet.readSenseCount(): > 100 missing senses, suppressing messages
................................x
  0.247 seconds to process /Users/ar/workspace/sumo/WordNetMappings/cntlist with 37000 senses.
INFO in WordNet.serialize(): origMaxNounSynsetID: 15300051 and: 15300051
WordNet.serialize(): WN has been serialized
ENTER DB.readSpreadsheet(/Users/ar/workspace/sumo/WordNetMappings/sentiment.csv, null)
ENTER DB.readSpreadsheet(java.io.FileReader@20c7642e, null)
EXIT DB.readSpreadsheet(java.io.FileReader@20c7642e, null)
  rows == [list of 8222 rows]
  0.08 seconds elapsed time
EXIT DB.readSpreadsheet(/Users/ar/workspace/sumo/WordNetMappings/sentiment.csv, null)
Error in OMWordnet.loadSerialized(): IOException is caught
java.io.FileNotFoundException: /Users/ar/workspace/KBs/WordNetMappings/omw.ser (No such file or directory)
	at java.io.FileInputStream.open0(Native Method)
	at java.io.FileInputStream.open(FileInputStream.java:195)
	at java.io.FileInputStream.<init>(FileInputStream.java:138)
	at java.io.FileInputStream.<init>(FileInputStream.java:93)
	at com.articulate.sigma.wordNet.OMWordnet.loadSerialized(OMWordnet.java:273)
	at com.articulate.sigma.wordNet.OMWordnet.readOMWfiles(OMWordnet.java:323)
	at com.articulate.sigma.KBmanager.setConfiguration(KBmanager.java:678)
	at com.articulate.sigma.KBmanager.initializeOnce(KBmanager.java:644)
	at com.articulate.sigma.KBmanager.initializeOnce(KBmanager.java:593)
	at com.articulate.sigma.trans.SUMOKBtoTPTPKB.main(SUMOKBtoTPTPKB.java:399)
INFO in OMWordnet.readOMWfiles(): reading files:
/Users/ar/workspace/sumo/OMW/als/wn-data-als.tab/Users/ar/workspace/sumo/OMW/arb/wn-data-arb.tab/Users/ar/workspace/sumo/OMW/bul/wn-data-bul.tab/Users/ar/workspace/sumo/OMW/cat/wn-data-cat.tab/Users/ar/workspace/sumo/OMW/cow/wn-data-cow.tab/Users/ar/workspace/sumo/OMW/dan/wn-data-dan.tab/Users/ar/workspace/sumo/OMW/ell/wn-data-ell.tab/Users/ar/workspace/sumo/OMW/eng/wn-data-eng.tab/Users/ar/workspace/sumo/OMW/eus/wn-data-eus.tab/Users/ar/workspace/sumo/OMW/fas/wn-data-fas.tab/Users/ar/workspace/sumo/OMW/fin/wn-data-fin.tab/Users/ar/workspace/sumo/OMW/fra/wn-data-fra.tab/Users/ar/workspace/sumo/OMW/glg/wn-data-glg.tab/Users/ar/workspace/sumo/OMW/heb/wn-data-heb.tab/Users/ar/workspace/sumo/OMW/hrv/wn-data-hrv.tab/Users/ar/workspace/sumo/OMW/isl/wn-data-isl.tab/Users/ar/workspace/sumo/OMW/ita/wn-data-ita.tab/Users/ar/workspace/sumo/OMW/ind/wn-data-ind.tab/Users/ar/workspace/sumo/OMW/jpn/wn-data-jpn.tab/Users/ar/workspace/sumo/OMW/nno/wn-data-nno.tab/Users/ar/workspace/sumo/OMW/nob/wn-data-nob.tab/Users/ar/workspace/sumo/OMW/pol/wn-data-pol.tab/Users/ar/workspace/sumo/OMW/por/wn-data-por.tab/Users/ar/workspace/sumo/OMW/qcn/wn-data-qcn.tab/Users/ar/workspace/sumo/OMW/spa/wn-data-spa.tab/Users/ar/workspace/sumo/OMW/swe/wn-data-swe.tab/Users/ar/workspace/sumo/OMW/tha/wn-data-tha.tab/Users/ar/workspace/sumo/OMW/zsm/wn-data-zsm.tabOMWordnet.serialize(): OMW has been serialized

INFO in KBmanager.setConfiguration(): SUMO
INFO in WordNet.termFormatsToSynsets(): changing origMaxNounSynsetID from: 15300051 to: 15300051
INFO in WordNet.termFormatsToSynsets(): result (orig,max): 15300051 and: 15300266
INFO in WordNet.serialize(): origMaxNounSynsetID: 15300051 and: 15300266
WordNet.serialize(): WN has been serialized
Info in KBmanager.initializeOnce(): completed initialization
KBmanager.serialize(): KBmanager has been serialized
Info in KBmanager.initializeOnce(): initialized is true
.Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 1 of relation altitude in formula:
(altitude ?OBJ1 ?OBJ2 ?HEIGHT)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 2 of relation altitude in formula:
(altitude ?OBJ1 ?OBJ2 ?HEIGHT)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 3 of relation altitude in formula:
(altitude ?OBJ1 ?OBJ2 ?HEIGHT)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 1 of relation altitude in formula:
(altitude ?OBJ1 ?OBJ2 ?HEIGHT)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 2 of relation altitude in formula:
(altitude ?OBJ1 ?OBJ2 ?HEIGHT)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 3 of relation altitude in formula:
(altitude ?OBJ1 ?OBJ2 ?HEIGHT)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 1 of relation altitude in formula:
(altitude ?OBJ1 ?OBJ2 ?Q)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 2 of relation altitude in formula:
(altitude ?OBJ1 ?OBJ2 ?Q)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 3 of relation altitude in formula:
(altitude ?OBJ1 ?OBJ2 ?Q)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 1 of relation superficialPart in formula:
(superficialPart ?PART ?OBJ)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 2 of relation superficialPart in formula:
(superficialPart ?PART ?OBJ)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 1 of relation superficialPart in formula:
(superficialPart ?ELEMENT ?OBJ)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 2 of relation superficialPart in formula:
(superficialPart ?ELEMENT ?OBJ)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 1 of relation superficialPart in formula:
(superficialPart ?ELEMENT ?OBJ)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 2 of relation superficialPart in formula:
(superficialPart ?ELEMENT ?OBJ)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 1 of relation superficialPart in formula:
(superficialPart ?S ?B)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 2 of relation superficialPart in formula:
(superficialPart ?S ?B)
...Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 1 of relation superficialPart in formula:
(superficialPart ?PART ?OBJ)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 2 of relation superficialPart in formula:
(superficialPart ?PART ?OBJ)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 1 of relation superficialPart in formula:
(superficialPart ?PART ?OBJ)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 2 of relation superficialPart in formula:
(superficialPart ?PART ?OBJ)
.Error in FormulaPreprocessor.findType(): no type information for predicate routeInSystem
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 1 of relation routeInSystem in formula:
(routeInSystem ?P ?S)
Error in FormulaPreprocessor.findType(): no type information for predicate routeInSystem
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 2 of relation routeInSystem in formula:
(routeInSystem ?P ?S)
.Error in FormulaPreprocessor.findType(): no type information for predicate maxValue
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 1 of relation maxValue in formula:
(maxValue ?R ?ARG ?N)
Error in FormulaPreprocessor.findType(): no type information for predicate maxValue
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 2 of relation maxValue in formula:
(maxValue ?R ?ARG ?N)
Error in FormulaPreprocessor.findType(): no type information for predicate maxValue
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 3 of relation maxValue in formula:
(maxValue ?R ?ARG ?N)
Error in FormulaPreprocessor.findType(): no type information for predicate minValue
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 1 of relation minValue in formula:
(minValue ?R ?ARG ?N)
Error in FormulaPreprocessor.findType(): no type information for predicate minValue
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 2 of relation minValue in formula:
(minValue ?R ?ARG ?N)
Error in FormulaPreprocessor.findType(): no type information for predicate minValue
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 3 of relation minValue in formula:
(minValue ?R ?ARG ?N)
Error in FormulaPreprocessor.findType(): no type information for predicate approximateDiameter
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 1 of relation approximateDiameter in formula:
(approximateDiameter ?Particle
  (MeasureFn ?Size Millimeter))
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 1 of relation superficialPart in formula:
(superficialPart ?SP ?O)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 2 of relation superficialPart in formula:
(superficialPart ?SP ?O)
.Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 1 of relation superficialPart in formula:
(superficialPart ?PART1 ?OBJ)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 2 of relation superficialPart in formula:
(superficialPart ?PART1 ?OBJ)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 1 of relation superficialPart in formula:
(superficialPart ?PART2 ?OBJ)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 2 of relation superficialPart in formula:
(superficialPart ?PART2 ?OBJ)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 1 of relation superficialPart in formula:
(superficialPart ?PART1 ?OBJ)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 2 of relation superficialPart in formula:
(superficialPart ?PART1 ?OBJ)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 1 of relation superficialPart in formula:
(superficialPart ?PART2 ?OBJ)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 2 of relation superficialPart in formula:
(superficialPart ?PART2 ?OBJ)
..Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 1 of relation superficialPart in formula:
(superficialPart ?OBJ3
  (HoleHostFn ?HOLE))
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 1 of relation superficialPart in formula:
(superficialPart ?OBJ3
  (HoleHostFn ?HOLE))
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 1 of relation superficialPart in formula:
(superficialPart ?OBJ3
  (HoleHostFn ?HOLE))
..Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 1 of relation superficialPart in formula:
(superficialPart ?COVER ?BODY)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 2 of relation superficialPart in formula:
(superficialPart ?COVER ?BODY)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 1 of relation superficialPart in formula:
(superficialPart ?COVER ?BODY)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 2 of relation superficialPart in formula:
(superficialPart ?COVER ?BODY)
...Error in FormulaPreprocessor.findType(): no type information for predicate shape
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 1 of relation shape in formula:
(shape ?OBJ Flat)
....Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 1 of relation superficialPart in formula:
(superficialPart ?PART ?OBJ2)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 2 of relation superficialPart in formula:
(superficialPart ?PART ?OBJ2)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 1 of relation superficialPart in formula:
(superficialPart ?PART ?OBJ2)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 2 of relation superficialPart in formula:
(superficialPart ?PART ?OBJ2)
.Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 1 of relation superficialPart in formula:
(superficialPart ?OBJ1 ?OBJ2)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 2 of relation superficialPart in formula:
(superficialPart ?OBJ1 ?OBJ2)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 1 of relation superficialPart in formula:
(superficialPart ?PART ?WHOLE)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 2 of relation superficialPart in formula:
(superficialPart ?PART ?WHOLE)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 1 of relation superficialPart in formula:
(superficialPart ?OBJ3 ?OBJ2)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 2 of relation superficialPart in formula:
(superficialPart ?OBJ3 ?OBJ2)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 1 of relation superficialPart in formula:
(superficialPart ?OBJ3 ?OBJ2)
Error in FormulaPreprocessor.computeVariableTypesRecurse(): no type information for arg 2 of relation superficialPart in formula:
(superficialPart ?OBJ3 ?OBJ2)

File written: /Users/ar/workspace/sumo/SUMO.tptp
```

**Comment by arademaker** [comment]
Minor problem with an unsolicited line break:

Error in WordNet.readNouns(): No match in /Users/ar/workspace/sumo/WordNetMappings/WordNetMappings30-noun.txt for line ElectricalConductor+

solved in 2b1604e

**Comment by arademaker** [comment]
I  believe we can close this issue. The original errors were solved.
